_docs/modules.md

```diff
@@ -0,0 +1,164 @@
+
+## Renaming Imports
+
+Sometimes, the module name can be long or ambiguous. To make the code more readable, you can rename the imported module using the `as` keyword:
+
+```glu
+import file as toto;
+
+func main() {
+    toto::function();
+}
+```
+In this example, `file` is renamed to `toto`, so you can access the `function()` using `toto::function()`. This is particularly useful when you have multiple modules with similar names or when you want to shorten the module name for convenience.
+
+## Importing Specific Functions
+
+If you only need a specific function from a module, you can import it directly:
+```glu
+import file::function;
+
+func main() {
+    function();
+}
+```
+By importing only `function()`, you don't need to reference the module name when calling the function. This can make the code cleaner, especially when working with frequently used functions.
+
+## Importing Multiple Items with Renaming
+
+You can also import multiple items from a module and rename them individually. This can be useful when you need to resolve naming conflicts or want to clarify the usage of certain functions:
+
+```glu
+import file::{function, function1 as function2};
+
+func main() {
+    function();
+    function2(); // This calls function1, but it's renamed to function2.
+}
+```
+In this case, `function()` is imported as is, but `function1()` is renamed to `function2()`. This allows you to use more descriptive or context-specific names in your code.
+
+## Importing Structs and Enums
+
+In addition to functions, Glu allows you to import structures (struct) and enumerations (enum) from other modules. This is particularly useful when you want to share data types between different parts of your project.
+
+### Importing a Struct
+Let's say you have a `Person` struct defined in a file named `types.glu`:
+
+```glu
+struct Person {
+    name: String,
+    age: Int,
+}
+```
+You can import this `Person` struct and use it in another file:
+
+```glu
+import types::Person;
+
+func main() {
+    let john: john = { "John", 25 }
+    std::print(john.name);
+}
+```
+
+Here, `Person` is imported from the `types` file, and the struct is instantiated and used in the `main` function.
+
+### Importing an Enum
+
+Enums are another important feature that can be imported using the `import` statement. Suppose you have the following enum defined in `status.glu`:
+
+```glu
+enum Status {
+    SUCCESS,
+    FAILURE,
+}
+```
+
+To use this enum in another file, you can import it like so:
+
+```glu
+import status::Status;
+
+func printStatus(value: Status) {
+    if value == Status::SUCCESS {
+        std::print("Success!");
+    } else {
+        std::print("Failure!");
+    }
+}
+```
+
+This functions uses the imported `Status` enum and checks whether it holds a Succes or a Failure value to print a status message.
+
```

Comment:
you should show that `import status::Status::SUCCESS` works too, so it becomes possible to just `value == SUCCESS`